frontend/src/app/libs/common-components-demo/demo-group/demo-group.component.scss

```diff
@@ -4,6 +4,7 @@
 
   &__container {
     padding: 72px 2%;
+    width: 50%;
```

Comment:
are you sure this update does not affect other demos?

Reply:
I think its shouldn't. it just a width of container which we put our components in.